src/libcmd/installables.cc

```diff
@@ -102,6 +102,28 @@ MixFlakeOptions::MixFlakeOptions()
         }}
     });
 
+    addFlag({
+        .longName = "reference-lock-file",
```

Comment:
"reference" seems more appropriate here, since the lock file may be completely foreign (e.g. I'm working on a project that uses `nixos-22.11` and want it to use my deployment's lock file, so I run `nix flake lock --reference-lock-file ~/deploy/flake.lock`). This is very much my personal opinion though and if there's broad agreement that it should be called "input" I'd be willing to change it.